src/Microsoft.AspNet.Mvc.TagHelpers/TagHelperContentExtensions.cs

```diff
@@ -51,7 +52,18 @@ public static class TagHelperContentExtensions
 
             using (var writer = new TagHelperContentWrapperTextWriter(encoding, content))
             {
-                RazorPage.WriteTo(writer, encoder, value, escapeQuotes: true);
+                using (var stringWriter = new StringWriter())
+                {
+                    RazorPage.WriteTo(stringWriter, encoder, value);
+
+                    // In this case the text likely came directly from the Razor source. Since the original string is
+                    // an attribute value that may have been quoted with single quotes, must handle any double quotes
+                    // in the value. Writing the value out surrounded by double quotes.
+                    //
+                    // This is really not optimal from a perf point of view, but it's the best we can do for right now.
```

Comment:
Remove this line. Same [reasons](https://github.com/aspnet/Razor/pull/615#discussion-diff-45298662) as the comment in aspnet/Razor#615.